packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -316,7 +321,9 @@ const helperBehavior = {
       prioritizedVaults.entriesPrioritizedGTE(quoteRatioPlusMargin);
     await (next ? facets.helper.liquidateAndRemove(next) : null);
 
-    facets.helper.reschedulePriceCheck();
+    trace('price check liq', next && next[0]);
```

Comment:
```suggestion
    trace('price check liq', next?.[0]);
```

Reply:
Hah: ```
Property 'next' does not exist on type '[string, KindFacet<{ getVaultSeat: ({ state }: MethodContext) => ZCFSeat; initVaultKit: ({ state, facets }: MethodContext, seat: ZCFSeat) => Promise<{ publicNotifiers: { ...; }; invitationMakers: { ...; } & RemotableBra```

Nope!

Reply:
Hmm, did you have exactly this line?
```
    trace('price check liq', next?.[0]);
```

When I check out your branch and make that edit it's golden.